modules/pref/src/main/Pref.scala

```diff
@@ -412,6 +415,16 @@ object Pref {
     )
   }
 
+  object TimelineEntries {
+    val ONLY_ME = 1
+    val ALL     = 2
+
+    val choices = Seq(
+      ONLY_ME -> "Never",
```

Comment:
`ONLY_MINE` would make more sense. Though why is it `ONLY_ME` and `ALL` in the code but "Never" and "Always" when shown to the user? Since there are only two options: "Show" and "Don't show", why not just use "Yes" and "No"? And also, there are already translations for this, both for "Always"/"Never" and "Yes"/"No".

Reply:
totally agreed, i made all those changes. that is, i went with the existing boolean choices YES/NO. thanks for the review! :)